wagtail/wagtailadmin/api/endpoints.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import, unicode_literals
+
+from collections import OrderedDict
+
+from wagtail.api.v2.endpoints import DocumentsAPIEndpoint, ImagesAPIEndpoint, PagesAPIEndpoint
+from wagtail.api.v2.filters import (
+    ChildOfFilter, DescendantOfFilter, FieldsFilter, OrderingFilter, SearchFilter)
+from wagtail.api.v2.utils import BadRequestError, filter_page_type, page_models_from_string
```

Comment:
If we're going to merge them both, it should be moved to wagtailcore, maybe it could be merged into the ``.type()`` queryset method somehow.